vector/v.db.select/main.c

```diff
@@ -5,12 +5,12 @@
  *
  * AUTHOR(S):    Radim Blazek
  *               OGR support by Martin Landa <landa.martin gmail.com>
- *               -f flag by Huidae Cho <grass4u gmail.com>
+ *               -n, -j, -f flags by Huidae Cho <grass4u gmail.com>
  *               group option by Luca Delucchi <lucadeluge gmail.com>
  *
  * PURPOSE:      Print vector attributes
  *
- * COPYRIGHT:    (C) 2005-2009, 2011-2014 by the GRASS Development Team
+ * COPYRIGHT:    (C) 2005-2009, 2011-2014, 2020 by the GRASS Development Team
```

Comment:
https://www.gnu.org/licenses/gpl-howto.en.html
> For software with several releases over multiple years, it's okay to use a range (“2008-2010”) instead of listing individual years (“2008, 2009, 2010”) if and only if every year in the range, inclusive, really is a “copyrightable” year that would be listed individually;

Then, for this module, would 2019 be considered a copyrightable year when there were no changes last year? I'm not a lawyer ;-).

Reply:
@neteler Any comments on this?

Reply:
I am totally fine with simplifying with 2005-2020 (i.e., use a range). No need to make it a (long) list of years.

Reply:
> I am totally fine with simplifying with 2005-2020 (i.e., use a range). No need to make it a (long) list of years.

Alright, in terms of copyrightable years, I would say the more the better. Let's use a range here so we include every single year between 2005 and 2020.